README.md

```diff
@@ -1 +1,50 @@
-This is the Videomap capstone project
+# Videomap
+[http://videomap-step-2020.appspot.com/](http://videomap-step-2020.appspot.com/)
+
+<img src="https://user-images.githubusercontent.com/36008213/91629325-ee36b280-e97c-11ea-9ec8-994a088b64a7.png" width=500>
+<img src="https://user-images.githubusercontent.com/55601789/91622207-dfcfa300-e94a-11ea-83a6-dd8b2095e58e.gif" width=500>
+<img src="https://user-images.githubusercontent.com/36008213/90663960-33314b00-e1ff-11ea-9790-fa5883098d59.png" width=500>
+
+## About
+Videomap is an interactive map that combines Google Maps and Youtube to provide a simple way for users to find videos related to location. Videomap includes additional features related to COVID-19 including:
+- statistical and graphical data related to the virus
+- routing with COVID-19 information
+- heat map functionality based on various COVID-19 data points (confirmed, active, deaths, recovered, per capita)
```

Comment:
I prefer more depths to each of these features. Feel free to copy from the design doc if there is any.
You can have Noe and Eyad to help elaborate these in follow-on pull requests.